hashicorp/dev-portal
# fix: update linux ubuntu/debian install script

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-leah-fixupdate-install-command-hashicorp.vercel.app/terraform/install#linux) 🔎
- [Support request](https://hashicorp.slack.com/archives/C5FSPUGDS/p1732651253085519) 🎟️

## 🗒️ What


This PR updates the Linux Ubuntu/Debian install script. It now matches the script in [this github issue](https://github.com/hashicorp/terraform/issues/32622#issuecomment-2494452268) and in the [packaging guide](https://www.hashicorp.com/official-packaging-guide)

## 🤷 Why



The current install script is not working properly.

## 📸 Design Screenshots


<img width="884" alt="Screenshot 2024-11-26 at 2 31 38 PM" src="https://github.com/user-attachments/assets/9aac40a4-c036-4a16-908c-a1189d828749">



## 🧪 Testing



1. Go to [preview link](https://dev-portal-git-leah-fixupdate-install-command-hashicorp.vercel.app/terraform/install#linux)
2. Verify the install script matches the one in the [packaging guide](https://www.hashicorp.com/official-packaging-guide) under the Add the HashiCorp repo section
